src/omotes_sdk/queue_names.py

```diff
@@ -14,22 +15,21 @@ def omotes_exchange_name() -> str:
         return "omotes_exchange"
 
     @staticmethod
-    def job_submission_queue_name(workflow_type: WorkflowType) -> str:
-        """Generate the job submission queue name given the workflow type.
+    def job_submission_queue_name() -> str:
+        """Generate the job submission queue name.
 
-        :param workflow_type: Workflow type.
         :return: The queue name.
         """
-        return f"job_submissions.{workflow_type.workflow_type_name}"
+        return "job_submissions"
 
     @staticmethod
-    def job_results_queue_name(job: Job) -> str:
+    def job_results_queue_name(job_uuid: uuid.UUID) -> str:
         """Generate the job results queue name given the job.
```

Comment:
Nitpicking: "Generate the job results queue name given the job **id**"

Reply:
Fixed in https://github.com/Project-OMOTES/omotes-sdk-python/pull/61/commits/c239f062aedb1ad419a44d4cd35fb65e2fa30806